src/main/scala/tilelink/TLLane.scala

```diff
@@ -14,7 +14,9 @@ abstract class TLLane8b10b(val clientEdge: TLEdgeOut, val managerEdge: TLEdgeIn)
     (implicit val c: SerDesConfig, implicit val b: BertConfig, implicit val m: PatternMemConfig, implicit val p: Parameters) extends Lane
     with HasEncoding8b10b
     with HasTLBidirectionalPacketizer
-    with HasTLController
+    with HasTLController {
+        override def allios = Seq(ssio, encoderio, decoderio, packetizerio) ++ debugio
+    }
```

Comment:
Hmm, I don't really like this. It's easy to mess up and requires a lot of boilerplate. An alternative might be to create a `GenericHasTLController` module, then make the current `HasTLController` implementation inherit from it so that this "API" doesn't change.